src/components/analytics/index.ts

```diff
@@ -1,12 +1,17 @@
 import ReactGA from 'react-ga'
 import { isMobile } from 'utils/userAgent'
 
+export const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
 const GOOGLE_ANALYTICS_ID: string | undefined = process.env.REACT_APP_GOOGLE_ANALYTICS_ID
+
+const storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
```

Comment:
how come we don't generate it here?

Reply:
it's just a randomly generated id right? we could technically use something like [nanoid](https://www.npmjs.com/package/nanoid), but i understand why we might want to just use GA's built in client id generation

Reply:
yeah we should use GA's format in case we ever decide to go down the cookies route :P